src/vs/workbench/parts/terminal/electron-browser/terminalLinkHandler.ts

```diff
@@ -100,18 +133,35 @@ export class TerminalLinkHandler {
 
 	protected get _localLinkRegex(): RegExp {
 		if (this._platform === platform.Platform.Windows) {
-			return WINDOWS_LOCAL_LINK_REGEX;
+			return this._winLocalLinkPattern;
 		}
-		return UNIX_LIKE_LOCAL_LINK_REGEX;
+		return this._unixLocalLinkPattern;
 	}
 
 	private _handleLocalLink(link: string): TPromise<void> {
 		return this._resolvePath(link).then(resolvedLink => {
 			if (!resolvedLink) {
 				return void 0;
 			}
-			const resource = Uri.file(path.normalize(path.resolve(resolvedLink)));
-			return this._editorService.openEditor({ resource }).then(() => void 0);
+
+			let normalizedPath = path.normalize(path.resolve(resolvedLink));
```

Comment:
Can this be pulled into a `formatLocalLinkPath` or similar named function that returns a URI in form `<path>#line,col`?